### DISCUSSION TITLE
Lara as a 'template'

### DISCUSSION THREAD

**Discussion by koesper** [primefaces/community#832] [PrimeNG Templates] [answered]
In the past, i've started various projects based on a premium template, since that immediately gives you a usable project structure and pages including header/footer/menu etc.

Are there plans to release the free Lara theme as a template as well?
This would be awesome, since i've bought the UI Kit as well, which is based on Lara

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately we don't have a plan for it but we're in the progress of making a new designer app that will help you to build custom templates easily.
